src/js/components/RangeInput/RangeInput.js

```diff
@@ -86,8 +95,9 @@ const RangeInput = forwardRef(
     // This is to make sure scrollbar doesn't move
     // when user changes RangeInput value.
     const handleMouseOver = () =>
-      setScroll({ x: window.scrollX, y: window.scrollY });
-    const handleMouseOut = () => setScroll({ x: null, y: null });
+      isScrollEnabled && setScroll({ x: window.scrollX, y: window.scrollY });
```

Comment:
It would be slightly more efficient to check before even assigning  the handler:
`onWheel={isScrollEnabled ? handleOnWheel : undefined}`

Reply:
Should I also prevent adding the event listener to the window if I'm switching to go this route? I was trying to make a change with the smallest footprint possible since I'm not familiar with the codebase or patterns within

Reply:
yes

Reply:
@Tbhesswebber just following up here -- are you willing to incorporate the changes suggested above?